### PR TITLE
fix: correct changelog for venting emitters

### PR DESCRIPTION
**Pull request by frodehk**
## Why is this pull request needed?

No v8-7.md file in docs. 

## What does this pull request change?

- [x] Create change file for v8.7

## Issues related to this change:
https://equinor-ecalc.atlassian.net/browse/ECALC-568?atlOrigin=eyJpIjoiNTIwMWNmM2JlZmQyNDQyYWIwYzY5NjFiYzI5YTczYWIiLCJwIjoiaiJ9

### CODE REVIEW

File: docs/docs/changelog/changelog.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## [8.7.0](https://github.com/equinor/ecalc/compare/v8.6.0...v8.7.0) (2023-12-05)
 
+### ⚠ BREAKING CHANGES
+
+* Change name from `DIRECT_EMITTERS` to `VENTING_EMITTERS` in input Yaml-file ([#303](https://github.com/equinor/ecalc/pull/303))
```
Comment (jsolaas) [inline docs/docs/changelog/changelog.md:7]: Is this file generated? Or why are we duplicating this in next?

**Review by jsolaas — APPROVED** [review]

File: docs/docs/changelog/changelog.md (hunk 1 above)
Comment (TeeeJay) [reply docs/docs/changelog/changelog.md:7]: this file is generated yes, we should not alter "that history". It was included in 8.6.0 afaik, should not be stating in 8.7.0 changes then

File: docs/docs/changelog/next.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ tags: [release, eCalc]
 sidebar_position: 1
 ---
 
-# eCalc
+# eCalc v8.7
```
Comment (TeeeJay) [inline docs/docs/changelog/next.md:9]: this should be in the `v8-7.md` file, as we have created that release..or should already have been in the 8.6 version, but we forgot to update the changelog. v8-7.md should exists, nevertheless...

Comment (TeeeJay) [reply docs/docs/changelog/next.md:9]: Next now, is to the potential v8.8 release

**Review by TeeeJay — APPROVED** [review]

File: docs/docs/changelog/changelog.md (hunk 1 above)
Comment (frodehk) [reply docs/docs/changelog/changelog.md]: The changelog.md is now rolled back, no change.

File: docs/docs/changelog/next.md (hunk 2 above)
Comment (frodehk) [reply docs/docs/changelog/next.md]: Things are updated now, with changes in the v8-7.md file.

**Pull request merged**